vignettes/teal-options.Rmd

```diff
@@ -84,10 +84,19 @@ This indicates whether to print the `JavaScript` console logs to the `R` console
 
 Default: `FALSE`.
 
-### `teal.renv.lockfile` (`character`)
+### `teal.lockfile.enable` (`logical`)
 
-The path to the pre-computed `renv` lockfile that will be shared through teal app. To read more about lockfile usage creation check `?teal::teal_lockfile`.
+This enables app users to download `renv` lockfile by clicking "download lockfile" in the footer. 
+If `teal.lockfile.path` is not specified then lockfile is created automatically by `teal`. 
```

Comment:
to me, this sentence belongs more into the `teal.lockfile.path` option description rather than `teal.lockfile.enable`

Reply:
This is difficult as lockfile-process is joint dependent on both options. Therefore I suggest to have a `teal.lockfile` section with both options enumerated within the section.

Reply:
Yes, I had this dilemma where to put what. You see, lockfile-process i joint-dependent on both options and it make sense to make `teal.lockfile` section instead and enumerate `teal.lockfile.path` and `teal.lockfile.enable` within this section. I decided not to make the separate section as the whole vignette is constructed in a way that each option has own section with `<option> <value class>` for example `teal.lockfile.path character`